package.json

```diff
@@ -3,7 +3,7 @@
   "name": "marklogic",
   "description": "The official MarkLogic Node.js client API.",
   "homepage": "https://github.com/marklogic/node-client-api",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
Should we make it 3.1.1 since there is no new functionality?

Reply:
If we release from develop, then 3.2.0 is fine, but if we weren't ready to release from develop, then yes, we should do 3.1.1 and toss this PR onto a branch off of master.